tests/Fake/FakeTodo.php

```diff
@@ -15,7 +15,7 @@ class FakeTodo
     /**
      * @var callable
      */
-    private $todoGet;
+    public $todoGet;
```

Comment:
privateのままでよさそうです。（composer test にパスすること確認済み）

```suggestion
    private $todoGet;
```